neqo-transport/src/connection/mod.rs

```diff
@@ -473,6 +473,7 @@ impl Connection {
             #[cfg(any(test, feature = "build-fuzzing-corpus"))]
             test_frame_writer: None,
         };
+        c.paths.set_pmtud(c.conn_params.pmtud_enabled());
```

Comment:
You could make this an argument to `Paths::new()` instead.  That might be sensible.